scripts-base/src/components/Example.ts

```diff
@@ -19,9 +20,7 @@ export class Example extends Component<ExampleData> {
 
```

Comment:
V rámci přepisu bychom mohli změnit i obsah `init()`, který obsahuje antipattern neukládání získaných referencí na elementy:
```js
this.getChild('.example-child', HTMLElement).innerText += ` ${this.data.name}!`
```
Didakticky lepší variantou:
```js
const child = this.getChild('.example-child', HTMLElement)
child.innerText += ` ${this.data.name}!`
```

Reply:
?!?
V čem je to antipattern, když se to prostě používá právě jednou? Tohle je úplně jedno, že to není v proměnné. :neutral_face:

Reply:
> Didakticky

Reply:
Co je didaktického na zavádění single-use proměnných, které ničemu nepřispívají?!

Reply:
Vytváření single-use proměnných nemusí být vždy na škodu. Souhlasím s @enzy, že tady by se to hodilo, protože `this.getChild('.example-child', HTMLElement)` s `.innerText += `${this.data.name}!`  je dost dlouhé a že se jedná o element `example-child` se v tom trochu ztrácí, takže jeho zvýraznění v podobě `const child` mi přijde jako dobrý nápad. Akorát teda IRL se to nebude jmenovat child, což tady nakonec vlastně mate. (Ale jako je to out of scope tohoto PR.)

Reply:
Whatever. Ty samotné komponenty se stejně budou řešit až separátně, tak si na to pak udělejte pull request, jestli chcete.